src/plugins/auto/auto_executable_network.cpp

```diff
@@ -26,13 +27,6 @@ void AutoExecutableNetwork::SetConfig(const std::map<std::string, IE::Parameter>
 }
 
 IE::Parameter AutoExecutableNetwork::GetConfig(const std::string& name) const {
-    {
-        std::lock_guard<std::mutex> lock(_autoSContext->_confMutex);
-        auto it = _autoSContext->_config.find(name);
-        if (it != _autoSContext->_config.end()) {
-            return it->second;
-        }
-    }
     IE_THROW(NotFound) << name << " not found in the ExecutableNetwork config";
```

Comment:
Is this logic used by MULTI too? @yangwang201911  Will we keep MULTI logic no changed? @wangleis

Reply:
@yangwang201911 @peterchen-intel We will keep the MULTI logic unchanged

Reply:
> @yangwang201911 @peterchen-intel We need to keep the MULTI logic unchanged

Why MULTI logic should be unchanged? Don't we need to align behavior across all meta plugins? Or I was missing something

Reply:
Need we align the same behavior between SetConfig and GetConfig?